composer.json

```diff
@@ -42,6 +42,9 @@
         "check-style": "phpcs -p --standard=PSR2 --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 src tests",
         "fix-style": "phpcbf -p --standard=PSR2 --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 src tests"
     },
+    "suggest": {
+        "mvdnbrk/postmark-inbound": "Allows you to process Postmark Inbound Webhooks.",
```

Comment:
That comma at the end is not valid JSON, which caused Travis (etc) to fail, could we get that removed, then I'll happily merge :) Thanks Mark!